src/lib.rs

```diff
@@ -42,17 +34,17 @@ pub struct SurfaceTexture {
 #[derive(Debug)]
 pub struct Pixels {
     // WGPU state
-    device: Rc<wgpu::Device>,
-    queue: Rc<RefCell<wgpu::Queue>>,
+    device: wgpu::Device,
+    queue: wgpu::Queue,
     swap_chain: wgpu::SwapChain,
     surface_texture: SurfaceTexture,
     present_mode: wgpu::PresentMode,
 
-    // List of render passes
-    renderers: Vec<BoxedRenderPass>,
+    // A default renderer to scale the input texture to the screen size
+    scaling_renderer: ScalingRenderer,
 
     // Texture state for the texel upload
-    texture: wgpu::Texture,
+    pixel_texture: wgpu::Texture,
```

Comment:
Why was this field renamed? It follows that other fields prefixed `texture` should have also adopted the new prefix for consistency. (These are all related.) But I still don't know what benefit we get by renaming this field.

Reply:
I wanted to provide an accessor for it, and I figured it makes sense to not just name it the generic `texture`. I can either remove the prefix, or add it to everything. Your call.

Reply:
I guess I prefer to keep the original name; this field is already a member of a struct named `Pixels`. The redundancy doesn't really add anything, IMHO.